packages/rn-tester-e2e/tests/helpers/utils.js

```diff
@@ -8,40 +8,42 @@
  * @format
  */
 
-import {driver} from '../../jest.setup';
-
 type PlatformsReference = {
   ios: string,
   android: string,
 };
 
 class Utils {
   async checkElementExistence(locator: string): Promise<boolean> {
-    await driver.$(locator).waitForDisplayed();
-    return driver.$(locator).isDisplayed();
+    await $(locator).waitForDisplayed();
```

Comment:
Nit: In all these helper methods, I suggest utilizing the `errorMsg` parameter in `waitForDisplayed()`. You can pass in the specific locator string so the error message is specific. If not, it will be confusing when debugging to see where it failed and which compnent.

For example,
await $(locator).waitForDisplayed({ timeout: 15000, timeoutMsg: "The element with locator=" + locator + " was not found"});

Reply:
It helps debugging a ton